### PR TITLE
Mongodb-Logging-Template

### PR DESCRIPTION
**Pull request by lchapman4**
Created a default Mongodb logging template. This will allow customers to easily send Mongodb logs to NR by simply renaming the .example file to .yml.

### CODE REVIEW

File: mongodb-log.yml.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+logs:
+# This sample file will forward mongodb error logs to NR once it's renamed to mongodb-log.yml
```
Comment (paologallinaharbur) [inline mongodb-log.yml.example:2]: could we follow the same style for the comment of https://github.com/newrelic/nri-redis/pull/134/files?

File: mongodb-log.yml.example [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+# On Linux systems no restart is needed
+  - name: "mongodblog"
+    file: /var/log/mongodb/mongodb.log
+    attributes:
+      logtype: mongodb
```
Comment (paologallinaharbur) [inline mongodb-log.yml.example:7]: missing newline at the end of the file

**Comment by lchapman4** [comment]
Thanks for catching that. I updated the file. Let me know if it's good now.

File: mongodb-log.yml.example (hunk 2 above)
Comment (lchapman4) [reply mongodb-log.yml.example]: Done. Let me know if I missed anything else.

File: mongodb-log.yml.example (hunk 1 above)
Comment (lchapman4) [reply mongodb-log.yml.example]: Done.

**Review by paologallinaharbur — APPROVED** [review]

**Pull request merged**